WebApplication/ClientApp/src/App.js

```diff
@@ -23,11 +23,12 @@ import './app.css';
 import Toolbar from './components/toolbar';
 import TabsContainer from './components/tabsContainer';
 import ProjectSwitcher from './components/projectSwitcher';
-import { fetchShowParametersChanged } from './actions/uiFlagsActions';
+import { showAdoptWithParamsFailed, setEnableEmbededMode, fetchShowParametersChanged } from './actions/uiFlagsActions';
```

Comment:
typo, should be `embedded` (several places to change)

Reply:
Fixed in last push as well